tor_ocr/main.py

```diff
@@ -76,8 +67,33 @@ def chunks(s, n):
         yield s[start:(start + n)]
 
 
-def run(config):
+# stolen from https://github.com/Zaargh/ocr.space_code_example/blob/master/ocrspace_example.py
+def ocr_space_url(url, overlay=False, api_key=os.getenv("OCR_API_KEY", "helloworld"), language='eng'):
+    """ OCR.space API request with remote file.
+        Python3.5 - not tested on 2.7
+    :param url: Image url.
+    :param overlay: Is OCR.space overlay required in your response.
+                    Defaults to False.
+    :param api_key: OCR.space API key.
+                    Defaults to environment variable "OCR_API_KEY", if it doesn't exist, it will use "helloworld"
+    :param language: Language code to be used in OCR.
+                    List of available language codes can be found on https://ocr.space/OCRAPI
+                    Defaults to 'en'.
+    :return: Result in JSON format.
+    """
 
+    payload = {'url': url,
```

Comment:
Do we know if this works with an Imgur gallery? More pre-processing may be needed.

Reply:
It does not work with Imgur links, only direct links to images. I'll add to the TODO for post processing, but I don't know if I'll do Imgur albums in this PR.